classes/classroom.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'student'
+
+class Classroom
+  attr_accessor :label, :students
```

Comment:
- [ ] I suggest that you make `students` an `attr_reader` only

Reply:
Thank you @DeliceLydia